pkg/executables/kubectl.go

```diff
@@ -1885,6 +1886,23 @@ func (k *Kubectl) GetBundles(ctx context.Context, kubeconfigFile, name, namespac
 	return response, nil
 }
 
+// GetEKSARelease gets an EKSARelease resource from a cluster.
+func (k *Kubectl) GetEKSARelease(ctx context.Context, kubeconfigFile, name, namespace string) (*releasev1alpha1.EKSARelease, error) {
```

Comment:
sorry I think i missed this
Where do we use it?

Reply:
cluster_manager uses it